src/dev.js

```diff
@@ -1,4 +1,4 @@
-const api = 'http://localhost:1337'
-//const api = 'https://api.campaigntracker.org'
+//const api = 'http://localhost:1337'
+const api = 'https://api.campaigntracker.org'
```

Comment:
You probably want to make this an environment-specific config variable or a flat out environment variable.